public/js/components/ButtonSettings.vue

```diff
@@ -15,7 +15,7 @@
 
     <label>
       Call Type
-      <select id="callTypeSelect" v-model="tile.callType" name="callType">
+      <select id="callTypeSelect" name="callType" v-model="tile.callType">
```

Comment:
```suggestion
      <select id="callTypeSelect" v-model="tile.callType" name="callType">
```

I’d like to revert this change as there is a linter rule from the ESLint plugin for Vue marking this order of attributes as a warning.

If you use an editor like VS Code, you should be able to see the warnings. Since this was introduced very recently, you might run `npm install` on your branch to update the dependencies.

In any case, don’t worry about this, this is not important.

Reply:
10-4, reverted back to original attribute order.